container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -4,8 +4,17 @@ set -x
 
 HOMEDIR=/var/lib/tempest
 TEMPEST_DIR=$HOMEDIR/openshift
-PROFILE_ARG=""
-CONCURRENCY="${CONCURRENCY:-}"
```

Comment:
You may want to deprecate the old org, but keep it functional for a while.

Reply:
Ok, understood. Fixed:)